manifests/claudie/ansibler.yaml

```diff
@@ -18,8 +18,41 @@ spec:
         app.kubernetes.io/name: ansibler
     spec:
       terminationGracePeriodSeconds: 1320
+      securityContext:
+        runAsUser: 1000
```

Comment:
How about further securityContext settings, e.g. not running in privileged mode, running as non-root, using read only root file system, etc ?

Reply:
That is a good point. I can look into that as well.